package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@northern/match",
+  "version": "1.0.0-alpha",
+  "main": "src/scss/_match.scss",
+  "style": "src/scss/_match.scss",
+  "sass": "src/scss/_match.scss",
+  "author": "Northern Commerce",
```

Comment:
Shouldn't this be `Northern Commerce Inc.` or are we not including that?

Reply:
I think just `Northern Commerce` is fine.